lib/papertrail/http_client.rb

```diff
@@ -42,19 +42,48 @@ def get(path, params = {})
       if params.size > 0
         path = "#{path}?#{build_nested_query(params)}"
       end
-      on_complete(https.get(request_uri(path), @headers))
+      attempts = 0
+      begin
+        on_complete(https.get(request_uri(path), @headers))
+      rescue SystemCallError, StandardError => e
+        sleep 5.0
+        attempts += 1
+        retry if (e.class.name.start_with?('Errno::', 'Net::') and attempts < 3)
```

Comment:
I think we can remove the `e.class.name.start_with?('Errno::', 'Net::')` check and just do the `attempts < 3` check here.